crates/lib/src/deploy.rs

```diff
@@ -442,8 +459,38 @@ pub(crate) async fn pull(
     prog: ProgressWriter,
 ) -> Result<Box<ImageState>> {
     match prepare_for_pull(repo, imgref, target_imgref).await? {
-        PreparedPullResult::AlreadyPresent(existing) => Ok(existing),
+        PreparedPullResult::AlreadyPresent(existing) => {
+            // Log that the image was already present (Debug level since it's not actionable)
+            const PULL_JOURNAL_ID: &str = "5c4d3e2f1a0b9c8d7e6f5a4b3c2d1e0f9";
+            let msg = format!("Image already present: {}", imgref);
+            crate::journal::journal_send(
+                Priority::Debug,
+                &msg,
+                [
+                    ("MESSAGE_ID", PULL_JOURNAL_ID),
+                    ("BOOTC_IMAGE_REFERENCE", &imgref.image),
+                    ("BOOTC_IMAGE_TRANSPORT", &imgref.transport),
+                    ("BOOTC_STATUS", "already_present"),
+                ]
+                .into_iter(),
+            );
+            Ok(existing)
+        }
         PreparedPullResult::Ready(prepared_image_meta) => {
+            // Log that we're pulling a new image
+            const PULL_JOURNAL_ID: &str = "5c4d3e2f1a0b9c8d7e6f5a4b3c2d1e0f9";
```

Comment:
Why duplicate this `const`? Though it does seem to me that "no changes" and "pulling a new image" are *different* events